x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/MachineLearningFlyout/view.tsx

```diff
@@ -216,7 +246,7 @@ export function MachineLearningFlyoutView({
                   onClickCreate({ transactionType: selectedTransactionType })
                 }
                 fill
-                disabled={isCreatingJob || hasMLJob || isLoadingMLJob}
+                disabled={isCreatingJob || hasMLJob !== false || isLoadingMLJob}
```

Comment:
Is this change still necessary? Isn't `hasMlJob` always a boolean?

Reply:
oh whoops i missed this when i changed the return type of `getHasMLJob`